README.md

```diff
@@ -199,3 +199,7 @@ cargo test
 ```bash
 cargo build --release
 ```
+
+# Similar projects
+
+- [environ](https://github.com/double-labs/environ) Generate .env files from secrets stored in Google Cloud Storage
```

Comment:
I'd generalize to "stored in the cloud" to avoid this comment getting old, as adding backends is really fast and simple.

Reply:
done!